openapi.yaml

```diff
@@ -1,4 +1,4 @@
-openapi: 3.0.3
+openapi: 3.1.0
```

Comment:
@marcin-cb Hello, I'm revisiting this PR because this line break the Rust SDK code generator

```
  thread 'main' panicked at rust/build.rs:65:51:
  called `Result::unwrap()` on an `Err` value: UnexpectedFormat("invalid version: 3.1.0")
```

Rust client is generated with progenitor-middleware and it does not support 3.1.0, is this update necessary, can we roll it back?

Reply:
👋🏼 @juchiast main has updated the progenitor dep to permit 3.1.0, should not see this error anymore